src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you've set both `margin-top` and `margin-bottom` to `0`. Choose one direction to apply the margin to maintain consistency and prevent margin collapse issues.